app/views/admin_posts/_admin_post.html.erb

```diff
@@ -0,0 +1,39 @@
+<% # expects "admin_post" %>
```

Comment:
I pulled this out into a partial so it doesn't need to be repeated in both admin_posts/index.html.erb and admin_posts/show.html.erb